packages/main/src/Menu.ts

```diff
@@ -338,8 +389,15 @@ class Menu extends UI5Element {
 		}, MENU_OPEN_DELAY);
 	}
 
+	_isGroupParent(parent: MenuItem | Menu | MenuItemGroup): boolean {
```

Comment:
not used

Reply:
Yepp, I forgot to remove it after fixing Tsani's comments.